tests/pytest/oauth/test_views.py

```diff
@@ -137,6 +137,26 @@ def test_authorize_success_with_claim_false(
     assert result.url == reverse(ROUTE_CONFIRM)
 
 
+@pytest.mark.django_db
+@pytest.mark.usefixtures("mocked_analytics_module")
+def test_authorize_success_with_claim_error(
```

Comment:
This is good, but sort of misses the actual meat of this PR's change, which was to send the code along with analytics.

Reply:
Suggest injecting `mocked_analytics_module` directly and asserting the call on that.